src/mmw/js/src/modeling/templates/scenariosBar.html

```diff
@@ -10,5 +10,5 @@
 </div>
 <div class="fade-tabs"></div>
 <div class="compare-top-btn">
-    <a href="compare" class="btn btn-compare"><i class="fa fa-th"></i> Compare</a>
```

Comment:
I changed it to use `data-url` to avoid doing a full page reload and instead just go to the CompareController. What's the advantage of leaving it the way it is?